src/main/groovy/edu/oregonstate/mist/webapiskeleton/SkeletonApplication.groovy

```diff
@@ -0,0 +1,31 @@
+package edu.oregonstate.mist.webapiskeleton
+
+import edu.oregonstate.mist.api.Application
+import edu.oregonstate.mist.api.Configuration
+import io.dropwizard.setup.Environment
+
+/**
+ * Main application class.
+ */
+class SkeletonApplication extends Application<Configuration> {
```

Comment:
Is this class needed? This should just be part of the skeleton, but shouldn't be in your api.